test/Microsoft.AspNet.Mvc.TagHelpers.Test/OptionTagHelperTest.cs

```diff
@@ -169,7 +174,9 @@ public static IEnumerable<object[]> DoesNotUseViewContextDataSet
             // Assert
             Assert.Equal(
                 expectedOutput,
-                output.GenerateStartTag() + output.GenerateContent() + output.GenerateEndTag());
+                output.GenerateStartTag().ToString() +
+                output.GenerateContent().ToString() +
+                output.GenerateEndTag().ToString());
```

Comment:
nits:
- no need for `ToString()` with `GenerateStartTag` or `GenerateEndTag`
- use `GetContent()` with `GenerateContent`